src/js/main.js

```diff
@@ -227,8 +269,8 @@ Site.Shapes = {
   playAnimation: function() {
     var _this = this;
 
-    // Check is animating has been stopped; we do this here to fail faster
-    if (!_this.animating) {
+    // Check is animating has been stopped or is at limit; we do this here to fail faster
+    if (!_this.animating || _this.atLimit) {
```

Comment:
this line is not running when moving the map along one boundry. i.e. if you move all the way left then move up and down the left boundry the animation does not play and this function is never executed